sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/FunMatcher.kt

```diff
@@ -39,10 +39,14 @@ import org.jetbrains.kotlin.resolve.descriptorUtil.overriddenTreeUniqueAsSequenc
 
 private const val VARARG_PREFIX = "vararg ";
 
+private fun Set<String>.addOptional(optionalString: String?): Set<String> =
```

Comment:
How about calling the method `addIfNonEmpty`, making it clear on which condition it will be added?

Reply:
Fixed.